The software demo does not build

I am trying to build the demo project in the software dir as described in the README but it fails:

```Interactive
root@9517a373bc4c:/litex# git log -1 --oneline
041aa9bf (HEAD -> master, origin/master, origin/HEAD) soc/cores/clock/xilinx_us/USIDELAYCTRL: make sure sys clock domain is reseted when reference clock domain is reseted.
root@9517a373bc4c:/litex# litex_bare_metal_demo --build-path=build/arty/
Makefile:4: /software/common.mak: No such file or directory
make: *** No rule to make target '/software/common.mak'.  Stop.
cp: cannot stat 'demo/demo.bin': No such file or directory
```

I tested this in a plain docker image based on ubuntu to be sure I didn't have anything on my own system that I messed up. 

Essentially the dockerfile looks like this and above was just run interactively to show the errors more reabable.

```Dockerfile
FROM ubuntu:20.04
RUN apt-get update && apt-get install wget python3 -y
RUN apt-get update && apt-get install git build-essential -y
RUN apt-get update && apt-get install python3-setuptools -y
RUN wget https://raw.githubusercontent.com/enjoy-digital/litex/master/litex_setup.py
RUN chmod +x litex_setup.py
RUN ./litex_setup.py init install
RUN ./litex_setup.py gcc
RUN which litex_bare_metal_demo
RUN litex_bare_metal_demo --build-path=build/arty/
```

--- a/litex/soc/software/demo/README.md
+++ b/litex/soc/software/demo/README.md
@@ -6,8 +6,10 @@ This directory provides a minimal bare metal demo app that demonstrates how to e
 [> Build
 --------
 
-Imagine you just build the Arty example design from LiteX-Boards, to build the demo app, run:
- `$ litex_bare_metal_demo --build-path=build/arty/`
+Imagine you just built the Arty example design from LiteX-Boards. Build the demo app as follows; where the build path is the path to the your previously built Arty build directory:
+```
+litex_bare_metal_demo --build-path=build/arty/
+```
 
 [> Load
 -------

